src/adera/Machines/Rocket.cpp

```diff
@@ -183,10 +183,12 @@ void SysMachineRocket::update_physics(ActiveScene& rScene)
             (phys::constants::g_0 * machine.m_params.m_specImpulse);
         for (MachineRocket::ResourceInput const& resource : machine.m_resourceLines)
         {
+            const auto* pipe = resource.m_source.get_if<wiretype::Pipe>();
```

Comment:
check for null.

Unless somehow you know that you'll always get a non-null

in which case, use a reference instead.